src/backend/linux_raw/vdso_wrappers.rs

```diff
@@ -361,6 +361,8 @@ fn init() {
         let ptr = vdso.sym(cstr!("LINUX_2.6"), cstr!("__vdso_clock_gettime64"));
         #[cfg(target_arch = "mips64")]
         let ptr = vdso.sym(cstr!("LINUX_2.6"), cstr!("__vdso_clock_gettime"));
+        #[cfg(target_arch = "loongarch64")]
+        let ptr = vdso.sym(cstr!("LINUX_5.19"), cstr!("__vdso_clock_gettime"));
```

Comment:
This is wrong, LoongArch vDSO symbol version is `LINUX_5.10` [according to the sources](https://github.com/torvalds/linux/blob/master/arch/loongarch/vdso/vdso.lds.S). (It's the version at the time LoongArch upstream work has started, not the time when the port was mainlined, IIUC.)